packages/matter.js/src/ble/BTPSessionHandler.ts

```diff
@@ -0,0 +1,186 @@
+/**
```

Comment:
I'd name this file `BtpSessionHandler.ts`

Reply:
done